wpeverest/user-registration
# Enhance/pop up

### All Submissions:






### Changes proposed in this Pull Request:

A pop up with links to our documentations is shown (only once) when a user creates a new form after plugin activation. An icon which toggles `Quick Links` box is also added in the form builder. In the `Quick Links` box, links to our documentations and support links are added.



### How to test the changes in this Pull Request:

1. Deactivate and re-activate `User Registration` plugin
2. Create new form
3. When you create a new form a pop up will be shown, which contains links to docs
4. Now, try to create another form. The pop should not be shown now.
5. An icon to toggle `Quick Links` box should be present at the right-bottom position of the form builder.
6. Try to toggle it and see if everything is working properly.

### Types of changes:




### Other information:




### Changelog entry

> Enhance - A `Quick Links` box and a pop up with links to our docs are added
